webapp/src/main/webapp/news-list-view/components/views/NewsLatestView.vue

```diff
@@ -83,6 +110,7 @@ export default {
   },
   mounted() {
     this.$nextTick().then(() => this.$root.$emit('application-loaded'));
+    this.hasSmallWidthContainer = (this.$refs.newsLatestView?.clientWidth *100 / window.screen.width) < 33;
```

Comment:
Why are you using this check ? couldn't we use Mobile vs Tablet vs Desktop?

Reply:
this method allows to detect the dimension of the screen and to check if the display of the mosaic template is by column or by the normal form.